app/assets/stylesheets/_navigation.css.scss

```diff
@@ -1,26 +1,27 @@
 // Navigation Styles
 
-.navbar-home {
-  @extend .navbar-brand;
-  font-size: inherit;
+.navbar-right > li > a {
+  padding-right: 5px;
 }
 
-nav {
-  @extend .navbar;
-  @extend .navbar-default;
-}
-
-.navbar-collapse-button {
-  @extend .navbar-toggle;
-  @extend .collapsed;
-}
+.navbar-nav > li.gravatar {
+  margin-right: -5px;
+  .round {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3